packages/nitro-protocol/package.json

```diff
@@ -57,9 +57,6 @@
     "chai": "^4.3.6",
     "dotenv": "^14.3.2",
     "eslint": "7.17.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-prettier": "3.3.1",
```

Comment:
Why remove these?

Reply:
Not sure this was actually needed. Reverted in https://github.com/statechannels/go-nitro/pull/1641/commits/c11c846221f5bddc6aad77cee5be1a5dc2af643d